assets/template.go

```diff
@@ -55,10 +55,22 @@ type TemplateComponent interface {
 	Params() []TemplateParam
 }
 
+// TemplateStatus is the status for the template translation
+type TemplateStatus string
+
+// different status that template translations can have
+const (
+	TemplateStatusApproved    TemplateStatus = "A"
+	TemplateStatusPending     TemplateStatus = "P"
+	TemplateStatusRejected    TemplateStatus = "R"
+	TemplateStatusUnsupported TemplateStatus = "X"
+)
```

Comment:
We don't use single char codes in the engine... 

Maybe this could be just `Approved() bool` since that's all that matters to the engine

Reply:
Okay I can change that